launch_ros/launch_ros/actions/node.py

```diff
@@ -61,8 +63,11 @@ def __init__(
         self, *,
         node_executable: SomeSubstitutionsType,
```

Comment:
Maybe, change this one to executable too?

Reply:
Sure, I don't mind doing this, but it will trigger a lot more changes on dependent packages.

Also, I noticed that the front-end uses slight different names for this and for the package name: `exec` and `pkg`. I don't think anything needs to be done about it, but just thought I'd point it out since the discrepancies between Python and the front-end are what motivated the current changes I'm making.

Reply:
> Sure, I don't mind doing this, but it will trigger a lot more changes on dependent packages.

> Also, I noticed that the front-end uses slight different names for this and for the package name: exec and pkg. I don't think anything needs to be done about it, but just thought I'd point it out since the discrepancies between Python and the front-end are what motivated the current changes I'm making.

Yeah, that's true. I think that have some arguments prefixed with `node_` and some others not, is strange. I would avoid the `node_` prefix everywhere, though it can be split in different PRs to avoid handling an annoing number of PRs at the same time.

About the difference between the frontend and python, I prefer the short versions for the frontend (`pkg`, `exec`), as the other ones are much more verbose. I also think that the current Python names are ok.

Reply:
I'll open a follow-up set of PRs deprecating `node_executable`.

Reply:
I finally got around to doing this. Here's the PR: https://github.com/ros2/launch_ros/pull/140